src/EtherFiNodesManager.sol

```diff
@@ -358,6 +359,10 @@ contract EtherFiNodesManager is
             phase: IEtherFiNode(etherfiNode).DEPRECATED_phase()
         });
         IEtherFiNode(etherfiNode).migrateVersion(_validatorId);
+
+        if (_validatorId != 0) {
```

Comment:
this is the fix for the bug I found.

Without it, the `fullWithdraw` of the validator with its safe version 0 will fail